modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/Button.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton, {ClayButtonWithIcon} from '@clayui/button';
+import React from 'react';
+
+export default function Button({
+	componentId: _componentId,
+	cssClass,
+	label,
+	locale: _locale,
+	portletId: _portletId,
+	portletNamespace: _portletNamespace,
+	...otherProps
+}) {
+	const {icon} = {...otherProps};
```

Comment:
Can't we receive this as a regular prop instead extracting it from `otherProps`?

Reply:
Sure I can look into that

Reply:
done :heavy_check_mark: